### PR TITLE
docs: class diagramm

### PR DESCRIPTION
**Pull request by TheMaxens**
I created a first draft of a class diagram to illustrate the data model of a ball bearing. 

Is something wrong or missing?

I omitted the parameter a2 or xB2 because there is only one distance per bearing.

### CODE REVIEW

**Review by philippabele — CHANGES_REQUESTED** [review]

File: docs/include/content.tex [hunk 1]
```diff
@@ -30,6 +30,34 @@ \chapter{Requirements}
...
+                
+                
+            \end{class}
+        \end{tikzpicture}
+
+        \begin{tikzpicture}
+            \begin{class}[text width=8cm]{OArrangement}{0,0}
+                \attribute{- bearingA : Bearing}
+                \attribute{- bearingB : Bearing}
+                \attribute{- Fr : double}
```
Comment (philippabele) [inline docs/include/content.tex:51]: Fr & Fa and their corresponding geometric paramters are loads on the shaft-bearing-system. May it be better to seperate them in a class "Load"?

n, the rotational speed, belongs to the load class as well.

**Review by philippabele — CHANGES_REQUESTED** [review]

File: docs/include/content.tex [hunk 2]
```diff
@@ -30,6 +30,34 @@ \chapter{Requirements}
             \path [connector] (result) -- (end);
         \end{tikzpicture}
 
+        \begin{tikzpicture}
+            \begin{class}[text width=8cm]{Bearing}{0,0}
+                \attribute{- Cdyn : double}
+                \attribute{- Y : double}
+                \attribute{- e : double}
+                \attribute{- xB1 : double}
+                \attribute{- xD1 : double}
```
[the remark this answers is not in the document]
Comment (TheMaxens) [reply docs/include/content.tex:39]: Will there be other shaft bearing systems in addition to the O-arrangement? 
Should I create a class "shaft-bearing-system" or is the O-arrangement class enough?

File: docs/include/content.tex (hunk 1 above)
Comment (TheMaxens) [reply docs/include/content.tex:51]: It seems logical, I will create the class "Load".

File: docs/include/content.tex (hunk 2 above)
[the remark this answers is not in the document]
Comment (philippabele) [reply docs/include/content.tex:39]: O-arrangement ist fine for this project.

**Comment by TheMaxens** [comment]
I created the class Load with the corresponding parameters. Additionally I put the parameter xD1 and xD2 to the O-arrangement class. 

I added a parameter load (of type Load) to the bearing-class. Is this ok or should I add it to the O-arrangement class?

**Comment by philippabele** [comment]
> I created the class Load with the corresponding parameters. Additionally I put the parameter xD1 and xD2 to the O-arrangement class.
> 
> I added a parameter load (of type Load) to the bearing-class. Is this ok or should I add it to the O-arrangement class?

Adding the Load to the O-arrangement sounds better. The "Load" on the shaft-bearing-system will be transformed into "P" an equivalent Load with respect to a single bearing.

By the way, where is the Parameter P in the bearing class ?

**Comment by TheMaxens** [comment]
Ah you're right. I forgot to add all the interim results (a, b, c, radiale Lagerkraft, usw.). 
I will append it.

**Comment by TheMaxens** [comment]
I added the following parameters to the "Bearing" class:
- P
- Fr
- Fa
- lh10

I added the following parameters to the "O-arrangement" class:
- load
- a
- b
- c
- lh10	(will be the final resullt)

I think adding the interim results a, b and c to the O-arrangement class makes sense because they refer to the whole system.

**Review by philippabele — APPROVED** [review]

**Pull request merged**